documentation/how-to/scaling.md

```diff
@@ -26,11 +26,21 @@ deployed as a separate application which can then be connected together by
 integrating with the same database. These services can then be scaled according
 to your needs.
 
+Note:
+
+- In a scaled deployment, only the unit deployed with the service `frontend`
+  needs to be related to the `nginx-ingress-integrator` charm for ingress and
+  the `openfga-k8s` charm for authorization.
+- The unit deployed with the service `frontend` should also handle the
+  `internal-frontend` service, which is the internal gateway that the `worker`
+  service uses to bypass authorization rules imposed for external connections.
+  This is only necessary when authorization is enabled.
+
 To deploy the services separately in a scalable way, you must deploy the
 application as follows:
 
 ```bash
-juju deploy temporal-k8s --config services="frontend"
+juju deploy temporal-k8s --config services="frontend,internal-frontend"
```

Comment:
If these two components will be deployed together even in a scaled deployment, could they be wrapped up in just `frontend`?

Reply:
Good idea, i've modified it such that both services will be enabled once `frontend` is enabled as a service. Thanks!